clang/lib/Format/UnwrappedLineFormatter.cpp

```diff
@@ -532,6 +538,20 @@ class LineJoiner {
       }
       return MergedLines;
     }
+
+    // Previously, UnwrappedLineParser would move the left brace to a new line
+    // when AllowShortFunctionBodiesOnASingleLine is enabled. However, if the
+    // function body cannot fit on a single line, and Style.BraceWrapping.AfterFunction
+    // is false, we should merge the function name and the left brace back onto
+    // the same line
```

Comment:
```suggestion
    // the same line.
```
Comments end in full stop.

Reply:
Thanks.